modules/hikari/src/main/scala/doobie/hikari/Config.scala

```diff
@@ -19,15 +19,15 @@ import scala.concurrent.duration.Duration
 /** Configuration case class, susceptible to PureConfig.
   * Helps with creating `com.zaxxer.hikari.HikariConfig`,
   * which in turn is used to create `doobie.hikari.HikariTransactor`.
-  * See the method `HikariTransactor.fromConfig` */
+  * See the method `HikariTransactor.fromConfigAutoEc` */
 final case class Config(
   catalog: Option[String] = None,
   connectionTimeout: Duration = Duration(30, TimeUnit.SECONDS),
   idleTimeout: Duration = Duration(10, TimeUnit.MINUTES),
   leakDetectionThreshold: Duration = Duration.Zero,
-  maximumPoolSize: Option[Int] = None,
+  maximumPoolSize: Option[Int] = Some(10),
   maxLifetime: Duration = Duration(30, TimeUnit.MINUTES),
-  minimumIdle: Option[Int] = None,
+  minimumIdle: Option[Int] = Some(10),
```

Comment:
If these values are already defaulted to 10 in hikari, do we need to set them explicitly here?

Reply:
Nope, but it makes it more user friendly. Users will immediately see what are the default values that they will get. Not doing so would be more obscure and users would have to do more investigation to see what will actually happen.

Reply:
Gotcha thanks!